.test-infra/jenkins/job_beam_ListView_Create.groovy

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This creates a standard List View for Beam. It filters out executors not associated with Beam
+// and pulls in all jobs which start with 'beam_'.
+listView('Beam') {
+    description('Jenkins builds for the Beam project.')
+    filterBuildQueue()
+    filterExecutors()
+    jobs {
+        regex(/beam_.+/)
+    }
+    jobFilters {
+        all()
+    }
+    columns {
+        status()
+        weather()
+        name()
+        lastSuccess()
+        lastFailure()
+        lastDuration()
+        buildButton()
```

Comment:
https://jenkinsci.github.io/job-dsl-plugin/#path/listView-columns doesn't list the ability to have the last success description. Have added console.